command/agentproxyshared/cache/static_secret_capability_manager_test.go

```diff
@@ -299,9 +299,12 @@ func TestSubmitWorkUpdatesIndex(t *testing.T) {
 
 	newIndex, err := sscm.leaseCache.db.GetCapabilitiesIndex(cachememdb.IndexNameID, indexId)
 	require.Nil(t, err)
+	newIndex.IndexLock.RLock()
+	readablePaths := newIndex.ReadablePaths
+	newIndex.IndexLock.RUnlock()
```

Comment:
It looks like StartRenewingCapabilities may mutate the contents of the map as well, so I think we need to hold onto the lock long enough to copy or compare the map.

Reply:
Oh, whoops, you're completely right haha. Fixed!